configurations/rolling-upgrade-artifacts.yaml

```diff
@@ -0,0 +1,6 @@
+# workloads
+stress_before_upgrade: cassandra-stress write cl=QUORUM n=10000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=200 -pop seq=1..10000000 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5
+stress_during_entire_upgrade: cassandra-stress read cl=QUORUM duration=25m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=200 -pop seq=1..10000000 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5
+stress_after_cluster_upgrade: cassandra-stress read cl=QUORUM duration=10m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=200 -pop seq=1..10000000 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5
```

Comment:
This shouldn't be "duration=" but "n=".
Also, if we adjust to my comments above, it should run on both keyspace.

Reply:
Changed